common/djangoapps/util/course.py

```diff
@@ -76,7 +76,15 @@ def has_certificates_enabled(course):
     return course.cert_html_view_enabled
 
 
-def should_display_grade(end_date):
-    if end_date and end_date < now().replace(hour=0, minute=0, second=0, microsecond=0):
-        return True
-    return False
+def should_display_grade(course_overview):
+    """
+    Returns True or False depending upon either certificate available date
+    or course-end-date
+    """
+    course_end_date = course_overview.end_date
+    cert_available_date = course_overview.certificate_available_date
+    current_date = now().replace(hour=0, minute=0, second=0, microsecond=0)
+    if cert_available_date:
+        return True if cert_available_date < current_date else False
```

Comment:
You can simply return `return cert_available_date < current_date`, no need to specify if else. It will return a boolean based on the comparison.

Reply:
Thanks for pointing out, Resolved!